rootfs/etc/profile.d/Ω_overrides.sh

```diff
@@ -0,0 +1,10 @@
+## Load user's custom overrides
+OVERRIDE_LIST=()
```

Comment:
```suggestion
SOURCE_OVERRIDE_LIST=()
```

Reply:
Also, "override" seems to imply we need to override something. I think it could just be `SOURCE_LIST` since that is what it is. It might be adding or overriding functionality.

Reply:
The variable will contain the list of "override" files to source. It is a parallel construction to `PREFERENCES_LIST`. Plus it is internal and documented. I don't want to change that name.

Reply:
![image](https://user-images.githubusercontent.com/52489/54899048-be33d780-4e8b-11e9-913c-08e55be047a6.png)


Where does the `PREFERENCES_LIST` convention come form?

Reply:
I made changes. See 
https://github.com/cloudposse/geodesic/pull/422/commits/dfadc0ed0a57f3114f119a917776b6efe00a1907